widgets/BigQueryExecute-action.json

```diff
@@ -193,6 +193,63 @@
           "name": "serviceAccountJSON"
         }
       ]
+    },
+    {
+      "label": "Retry Configuration",
+      "properties": [
+        {
+          "widget-type": "toggle",
+          "label": "Retry On Backend Error",
+          "name": "retryOnBackendError",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "YES"
+            },
+            "off": {
+              "value": "false",
+              "label": "NO"
+            },
+            "default": "false"
```

Comment:
shouldn't this be set to `true` by default, @tivv @fernst wdyt??
I thought aim is to add some resilency to the plugin to reduce intermittent failures but if the behavior is `false` by default, users will need to enable it explicitly and modify every pipeline.

Reply:
Your point makes a lot of sense. Setting this behavior to true by default aligns with the goal of enhancing plugin resilience and reducing intermittent failures.

Reply:
So, we usually have two things:
 * Default for UI. This is what we want default experience to be. In this case it should be "true"
 * Default for backend. This is usually how things worked before to ensure backward compatibility unless we have explicit OK from product to break backward compatibility

Reply:
@itsankit-google we had a discussion with deepinder, the UI fields should be hidden and true by default !